java/cdk/src/main/java/sleeper/cdk/stack/bulkimport/EmrStudioStack.java

```diff
@@ -0,0 +1,158 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sleeper.cdk.stack.bulkimport;
+
+import software.amazon.awscdk.NestedStack;
+import software.amazon.awscdk.RemovalPolicy;
+import software.amazon.awscdk.services.ec2.ISecurityGroup;
+import software.amazon.awscdk.services.ec2.IVpc;
+import software.amazon.awscdk.services.ec2.Peer;
+import software.amazon.awscdk.services.ec2.Port;
+import software.amazon.awscdk.services.ec2.SecurityGroup;
+import software.amazon.awscdk.services.ec2.Vpc;
+import software.amazon.awscdk.services.ec2.VpcLookupOptions;
+import software.amazon.awscdk.services.emr.CfnStudio;
+import software.amazon.awscdk.services.emr.CfnStudioProps;
+import software.amazon.awscdk.services.iam.Effect;
+import software.amazon.awscdk.services.iam.IRole;
+import software.amazon.awscdk.services.iam.ManagedPolicy;
+import software.amazon.awscdk.services.iam.ManagedPolicyProps;
+import software.amazon.awscdk.services.iam.PolicyDocument;
+import software.amazon.awscdk.services.iam.PolicyStatement;
+import software.amazon.awscdk.services.iam.PolicyStatementProps;
+import software.amazon.awscdk.services.iam.Role;
+import software.amazon.awscdk.services.iam.RoleProps;
+import software.amazon.awscdk.services.iam.ServicePrincipal;
+import software.amazon.awscdk.services.s3.BlockPublicAccess;
+import software.amazon.awscdk.services.s3.Bucket;
+import software.amazon.awscdk.services.s3.BucketEncryption;
+import software.amazon.awscdk.services.s3.IBucket;
+import software.constructs.Construct;
+
+import sleeper.cdk.Utils;
+import sleeper.configuration.properties.instance.InstanceProperties;
+
+import java.util.List;
+import java.util.Locale;
+
+import static sleeper.configuration.properties.instance.CdkDefinedInstanceProperty.BULK_IMPORT_EMR_SERVERLESS_STUDIO_BUCKET;
+import static sleeper.configuration.properties.instance.CdkDefinedInstanceProperty.BULK_IMPORT_EMR_SERVERLESS_STUDIO_URL;
+import static sleeper.configuration.properties.instance.CommonProperty.ID;
+import static sleeper.configuration.properties.instance.CommonProperty.SUBNETS;
+import static sleeper.configuration.properties.instance.CommonProperty.VPC_ID;
+
+/**
+ * An {@link EmrStudioStack} creates an SQS queue that bulk import jobs can be sent to. A message
+ * arriving on this queue triggers a lambda. That lambda creates an EMR cluster that executes the
+ * bulk import job and then terminates.
+ */
+public class EmrStudioStack extends NestedStack {
+    private ISecurityGroup defaultEngineSecurityGroup;
+    private ISecurityGroup workspaceSecurityGroup;
+    private IBucket studioBucket;
+
+    public EmrStudioStack(Construct scope, String id, InstanceProperties instanceProperties) {
+        super(scope, id);
+        String instanceId = instanceProperties.get(ID);
+
+        studioBucket = Bucket.Builder.create(this, "EmrStudioBucket")
```

Comment:
Do we need a specific bucket for this given we already create a bucket for bulk import?

Reply:
I have updated to use the bulk import bucket